playbooks/reboot_cluster.yml

```diff
@@ -14,6 +14,14 @@
               delay=15
     delegate_to: "{{ groups['controller']|first }}"
 
+  - name: Ensure Galera is up before rebooting next node
+    wait_for: port=4567
+
+  - name: Ensure mysql is running
+    service:
+      name: mysql
+      state: started
```

Comment:
Probably want something in here to check Neutron agent status as well before continuing.